src/classes/PS_Request.cls

```diff
@@ -198,10 +215,10 @@ public inherited sharing class PS_Request {
         * @param commitmentId Recurring Donation Commitment Id (required for cancel commitment endpoint only)
         * @return String Full endpoint
         */
-        private String getEndpoint(ElevateEndpoint endpoint, String commitmentId) {
+        public String getEndpoint(ElevateEndpoint endpoint, String commitmentId) {
             String value;
             if (endpoint == ElevateEndpoint.TOKENIZE) {
-                value = '/v1/payments/public/tokens';
+                value = '/v1/payments/verified/tokens';
```

Comment:
Does it make sense to move all endpoint URL's to constants at the top of the class so that they're easier to find/review/change in the future?

Reply:
It has been done.
@rponti-sforg , Cancel Commitment needs to be retested (happy path).